README.md

```diff
@@ -68,6 +69,35 @@ You can jump directly to "*Available apps, installation procedure*"
 
 Ansible
 -------------------------
+### Coreos on GCE
```

Comment:
Furthermore we've have a lot of different use cases
* calico bgp can be documented more precisely
* opencontrail configuration
* weave
* CoreOS
...